modules/ROOT/pages/install-validate.adoc

```diff
@@ -6,31 +6,26 @@ endif::[]
 
 To verify your installation and configuration:
 
-. If you are using JSON Web Tokens and not impersonation, verify the Named Credentials.
-.. Open *Setup*.
-.. Search for *Named Credentials*.
-.. Locate *anypoint.mulesoft* and click *Edit*.
-.. Verify that the list *External Data User Authentications* shows your administrative Anypoint user.
-. If you are using impersonation and not JSON Web Tokens, verify the Exchange External Data Source status is *Authenticated*.
-.. Open *Setup*.
-.. Search for *External Data Sources*.
-.. In the list select *Exchange*.
-.. Verify that the *Authentication Status* is *Authenticated*.
+. Open the Salesforce Application Launcher.
+. Use the search bar to find and open the *API Community Manager Diagnostics*.
+. Check the information about your Salesforce organization in the *Organization* section.
+.. Confirm that *Package Version* shows the version of your currently installed API Community Manager package.
+.. Confirm that *Connectivity with Anypoint* shows the URL your API community uses to access Anypoint and a green check.
 +
-image::validate-exchange-auth.png[Exchange Authenticated]
+If *Connectivity with Anypoint* does not show the URL and a green check, perform the xref:installation-prerequisites.adoc[prerequisite steps] and then repeat the installation.
+.. Confirm that *Cache* shows that the cache is configured.
 +
```

Comment:
Will it be clear to the audience what to look for to verify configuration (or not)?